### PR TITLE
Blog - Earthly Featured On star-history.com

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
@adamgordonbell [Preview Link](https://659694aa5984ee028554d0e6--ephemeral-narwhal-3ac3b2.netlify.app/blog/on-star-history/)

[Airtable link](https://airtable.com/appbtxxY2KaPio7dV/tblsY8AOw3qPBIUdZ/viwBflqOY7OlQqiNg/recn0FqutMQZFVEKK?blocks=hide)

**Comment by thtmnisamnstr** [comment]
Can we add a link to this under Earthly News on the blog home page, above "Earthly On devtools.FM"?

File: blog/_posts/2024-01-12-on-star-history.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Earthly Featured On star-history.com"
+categories:
+  - Tutorials
```
Comment (adamgordonbell) [inline blog/_posts/2024-01-12-on-star-history.md:4]: This should be category 'News'

Comment (msali123) [reply blog/_posts/2024-01-12-on-star-history.md:4]: Yes changing it.

**Comment by msali123** [comment]
[Preview link](https://6597013f993dc5256c843633--ephemeral-narwhal-3ac3b2.netlify.app/blog/on-star-history/) @adamgordonbell @thtmnisamnstr

**Comment by thtmnisamnstr** [comment]
lgtm

**Comment by thtmnisamnstr** [comment]
@adamgordonbell I know you merged this and the build ran successfully, but the blog post doesn't exist (https://earthly.dev/blog/on-star-history/) and it isn't linked under News on the blog homepage. Any idea what happened?

**Comment by adamgordonbell** [comment]
it's scheduled for Jan 12th.

I didn't pay much attention to the schedule date, because usually I'm fine with a slow roll out of posts.  But let me know if I should rename the file to change the date.

**Comment by thtmnisamnstr** [comment]
Jan 12 is fine. I didn't notice that. Just saw that it wasn't there.

**Pull request merged**